barretenberg/cpp/src/barretenberg/client_ivc/client_ivc.test.cpp

```diff
@@ -125,12 +125,13 @@ TEST_F(ClientIVCTests, Full)
     auto intermediary_acc = update_accumulator_and_decide_native(
         ivc.prover_fold_output.accumulator, kernel_fold_proof, foo_verifier_instance, kernel_vk);
 
-    VerifierFoldData kernel_fold_output = { kernel_fold_proof, function_vk_with_merge };
+    VerifierFoldData kernel_fold_output = { kernel_fold_proof, kernel_vk };
```

Comment:
kernel_vk is the same as function_vk_with_merge here but its the correct one name-wise